packages/frontier-core/cli/README.md

```diff
@@ -22,7 +22,25 @@ command using the syntax:
 ~$ frontier <COMMAND> <PARAMS> <FLAGS>
 ```
 
-&nbsp; &nbsp;
+&nbsp; &nbsp; &nbsp;
+
+<!-- custom-toc -->
+## Table of Contents
+
+* [About](#about)
+* [Installation & Usage](#installation--usage)
+* [Check Commands](#check-commands)
+* [Help Menu](#help-menu)
+* [Check Version](#check-version)
+* [Options](#options)
+<!-- custom-tocstop -->
+
+&nbsp; &nbsp; &nbsp;
+
+## About 
+
+Frontier offers a set of standard libraries and typing kits that can be use across engineering projects to allow for more consistency software development. With frontier, a set of plugins are available for use by developers which can be used to easily generate base project files contributing to a faster development process.  
```

Comment:
Incorrect about section?

Reply:
Am i to put this in the root folder README